build/Targets/VSL.Imports.targets

```diff
@@ -61,6 +61,19 @@
     </When>
   </Choose>
 
+  <!-- Because https://github.com/dotnet/roslyn/issues/7812 is not yet fixed, the IDE doesn't know if we set the PublicSign
+       flag. As a result, all design-time builds will thing we're real-signing, which causes semantics to get all screwed up.
+       The workaround for now is, for design-time builds only, to pass the DelaySign flag since that's "good enough". This
+       must be done in a target versus conditioning on BuildingProject, since BuildingProject itself is correctly set in a
+       target. -->
+  <Target Name="FixPublicSignFlagForDesignTimeBuilds" BeforeTargets="CoreCompile" Condition="'$(PublicSign)' == 'true'">
+    <PropertyGroup Condition="'$(BuildingProject)' == 'false'">
```

Comment:
Out of curiosity, why put this condition on the property group, rather than the target?